src/components/SiteHeader.tsx

```diff
@@ -29,8 +28,11 @@ const animationProps = {
 
 export const SiteHeader = (props: Props) => {
   const { pageId } = props;
-  const { accounts, requestAccounts, walletStatus } = useWeb3Context();
-  const shortAccounts = accounts.map(getShortAddress);
+
+  const { isSmall } = useResponsive();
+
+  if (isSmall) {
```

Comment:
Dead code or missing logic?

Reply:
Dead! Thanks. Gave up on implementing a burger menu when I realised it didn't play well with the wallet button. This is what's left of it. 

...Ill tidy all this up tomorrow, I initially just wanted to see how far we were from a mobile version and ended up figuring it out.